src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -414,9 +423,7 @@ static int32_t EnumAbbrevEraNames(const char* locale,
 
     char* localeNamePtr = localeNameBuf;
     char* parentNamePtr = parentNameBuf;
-
-    strncpy(localeNamePtr, locale, ULOC_FULLNAME_CAPACITY);
-    localeNamePtr[ULOC_FULLNAME_CAPACITY - 1] = 0;
+    STRING_COPY(localeNamePtr, sizeof(locale), locale, ULOC_FULLNAME_CAPACITY);
```

Comment:
Note that `sizeof(locale)` will give you the size of the *pointer* type, not the length of the string.
I *think* you maybe wanted something like `strlen(locale)` for this?

Reply:
Same with the above `sizeof(locale)` too.

Reply:
Yeah I think you're right, thanks for the hint.